docs/usage.rst

```diff
@@ -111,8 +111,27 @@ database handle (see `PDO::setAttribute`_).
 
     Set the default schema for the PDO connection.
 
+Custom Types
+============
 
+For creating a new connection please refer to `Connect to Crate`_.
+
+Insert custom types:
```

Comment:
Example of inserting a custom type (array) using the `bindValue()` method on the prepared statement.